pkg/server/api_plot.go

```diff
@@ -357,7 +357,9 @@ func makePlotsResponse(plotSeries map[string][]plot.Series, plotReq *PlotRequest
 					plotItem.Scale(plot.Value(scale))
 				}
 
-				seriesOptions[seriesItem.Name] = seriesItem.Options
+				if len(seriesItem.Options) != 0 {
```

Comment:
Assigning units to series manually is tedious. IMO, the connectors should gather the unit information from the backends. When you create a graph, the unit would then already be pre-filled with the appropriate unit provided by the connector.

However, rrd seems not to know anything about units at all. Collectd doesn't care either.
Influx can probably carry unit information, but needs to be filled by $other_statscollector.